math/ibex-lib/Portfile

```diff
@@ -0,0 +1,42 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           github 1.0
+PortGroup           waf 1.0
+PortGroup           compiler_blacklist_versions 1.0
+
+github.setup        ibex-team ibex-lib ibex-2.8.5
+categories          math
+platforms           darwin
+maintainers         nomaintainer
+license             LGPL-3.0
+
+description         IBEX is a C++ library for constraint processing over real numbers.
+long_description    IBEX is a C++ library for constraint processing over real numbers.\
+                    It provides reliable algorithms for handling non-linear constraints. In\
+                    particular, roundoff errors are also taken into account. It is based on\
+                    interval arithmetic and affine arithmetic. The main feature\
+                    of Ibex is its ability to build strategies declaratively\
+                    through the contractor programming paradigm. It can also be\
+                    used as a black-box solver.
+
+
+
+checksums           rmd160  6f1199be207c574433c610c504eb6e4a08ab02d8 \
+                    sha256  a55084bef36834a0d72f7b7fa02c491d0fb6a5c106c730f61e412c53cb977f73 \
+                    size    13201573
+
+
+compiler.blacklist-append {clang <= 800}
+
+depends_build       port:pkgconfig\
+                    port:python27
```

Comment:
python27? Can python37 be used instead?

Actually nothing in this portfile appears to actually set the build system to use Macports python at all. So is this even needed? Won't it pickup the existing macOS python? Check for a configure variable that points to the python executable and set it to use the Macports python.

Reply:
Unfortunately, no - the documentation specifies that Python3.x will not work... see http://www.ibex-lib.org/doc/install.html#requirements

Reply:
I still think the build system needs to be told to use Macports python. Won't the built in macOS python get used?